test/factories/trial_feedbacks.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :trial_feedback do
+    association :feedbackable, factory: :assignment
+    volunteer
```

Comment:
this one requires an after build method for the factory because otherwise, you're going to get three different volunteers, one for the assignment, the user and the trial_feedback each. But it should be the same.

```ruby
after(:build) do |trial_feedback|
  trial_feedback.volunteer: trial_feedback.assignment.volunteer
  trial_feedback.author: create(:user_volunteer, volunteer: trial_feedback.assignment.volunteer)
end
```

Reply:
So we checked so far, and this is not the case. The factory is called by the seeds and the tests, the volunteer, feedbackable and author set by the factory the are overwritten by them. (cla&tnt)